usaspending_api/common/custom_django_fields.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class NumericField(models.Field):
+    """default numeric type for Postgres"""
+
+    def db_type(self, connection):
+        return "numeric"
+
+
+class NaiveTimestampField(models.Field):
+    """timestamp column type without timezone"""
+
+    def db_type(self, connection):
+        return "timestamp without time zone"
+
+
+class BooleanFieldWithDefault(models.Field):
```

Comment:
I wonder if the issue you had with default value being ignored was fixed between 1.11 and 2.2. Tried to find something, but had no luck.

Reply:
Possibly. Haven't tested anything in Django 2.2 yet

Reply:
What was the problem?

Remember that Django defaults are only at the ORM level.  They aren't mirrored in the database as this is doing.  If you want defaults in the database, better to use an unmanaged model.  When we synced the database with the ORM a year or so ago, I had to add some SQL to the migrations to mimic defaults for some database columns that had them.

Doing it this way allows Django to support callable defaults (functions).

Reply:
The "problem" was the Django migration didn't set the default value to `false` which is how the Broker DB field is defined. Using the custom field was an easy approach to define the field in Djnago and allow migrations to occur with the desired end state. There is no discrepancy between the model and the table.  With the exception of the materialized views, all table schemas should be managed by django.

This is the resultant column in the table when using the custom field:
```is_active boolean NOT NULL DEFAULT false,```

Reply:
If we're copying the data from Broker, is having a default value even necessary?  Missing values would have already been converted to their defaults in their data.

Reply:
Probably not necessary.
Very true.

Rigorously matching the Broker's schema might cause issues down the road. We can weigh the pros and cons later if others devs have good ideas.